tests/integration/clientv3/maintenance_test.go

```diff
@@ -338,6 +341,52 @@ func TestMaintenanceSnapshotWithVersionVersion(t *testing.T) {
 	}
 }
 
+func TestMaintenanceSnapshotContentDigest(t *testing.T) {
+	integration2.BeforeTest(t)
+
+	clus := integration2.NewCluster(t, &integration2.ClusterConfig{Size: 1})
+	defer clus.Terminate(t)
+
+	populateDataIntoCluster(t, clus, 3, 1024*1024)
+
+	// reading snapshot with canceled context should error out
+	resp, err := clus.RandClient().SnapshotWithVersion(context.Background())
+	require.NoError(t, err)
+	defer resp.Snapshot.Close()
+
+	tmpDir := t.TempDir()
+	snapFile, err := os.Create(filepath.Join(tmpDir, t.Name()))
+	require.NoError(t, err)
+	defer snapFile.Close()
+
+	snapSize, err := io.Copy(snapFile, resp.Snapshot)
+	require.NoError(t, err)
+
+	// read the checksum
+	checksumSize := int64(32)
```

Comment:
Please use [sha256.Size](https://github.com/golang/go/blob/dc04f3ba1f25313bc9c97e728620206c235db9ee/src/crypto/sha256/sha256.go#L23) instead,

```suggestion
	checksumSize := int64(sha256.Size)
```

Reply:
Updated.